exts/cesium.omniverse/cesium/omniverse/bindings/CesiumOmniversePythonBindings.pyi

```diff
@@ -71,7 +71,7 @@ class ICesiumOmniverseInterface:
     def credits_start_next_frame(self) -> None: ...
     def get_asset_token_troubleshooting_details(self, *args, **kwargs) -> Any: ...
     def get_asset_troubleshooting_details(self, *args, **kwargs) -> Any: ...
-    def get_credits(self) -> List[Tuple[str, bool]]: ...
+    def get_credits(self) -> List[Tuple[str,bool]]: ...
```

Comment:
This is exactly what's failing in CI. There seems to be discrepancies with black in CI and in dev environments.

```
-    def get_credits(self) -> List[Tuple[str,bool]]: ...
+    def get_credits(self) -> List[Tuple[str, bool]]: ...
```

Reply:
Should be fixed in https://github.com/CesiumGS/cesium-omniverse/pull/684/commits/9cd8a8305be4f9b068b53913d247e6f481bd94eb. I had to mess with the `genStubs` script a bit to run `black` properly.